adjucency_list.rb

```diff
@@ -0,0 +1,81 @@
+require_relative 'linked_list'
+require_relative 'edge_list'
+require 'colorize'
+
+class AdjucencyList
+
+	attr_accessor :adj_list
+
+	def initialize(edge_list)
+		@edges_array = edge_list
+		biggest_id = find_biggest_id
+		@adj_list = Array.new(biggest_id+1) { LinkedList.new }
+		@person_lut = []
+		create_adj_list
+	end
+
+	def find_biggest_id
+		biggest_id = 0
+		@edges_array.each do |edge|
+			origin = edge[0]
+			destination = edge[1]
+			weight = edge[2]
+			if origin.id > biggest_id
+				biggest_id = origin.id
+			end
+			if destination.id > biggest_id
+				biggest_id = destination.id
+			end
+		end
+		biggest_id
+	end
+
+	def create_adj_list
+		@edges_array.each do |edge|
+			origin = edge[0]
+			destination = edge[1]
+			weight = edge[2]
+			x = origin.id
+			y = destination.id
+			@person_lut[x] = origin.name
```

Comment:
It is more readable to use `origin.id` and `destination.id` instead of `x` and `y`